dev/release/release_rc.sh

```diff
@@ -89,7 +89,6 @@ if [ "${RELEASE_SIGN}" -gt 0 ]; then
   gh run watch --repo "${repository}" --exit-status "${run_id}"
 
   # Create release candidate directory structure
-  rc_dir="apache-iceberg-go-${version}-rc${rc}"
```

Comment:
this variable is referenced by other commands

Reply:
oops from another change/manipulation i did. i'll fix that